assets/js/modules/analytics/common/account-create.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * AccountCreate component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, Fragment, useCallback, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../components/button';
+import ProgressBar from '../../../components/progress-bar';
+import { trackEvent } from '../../../util';
+import TimezoneSelect from './timzezone-select';
+import AccountField from './account-field';
+import PropertyField from './property-field';
+import ProfileField from './profile-field';
+import { STORE_NAME } from '../datastore/constants';
+
+import Data from 'googlesitekit-data';
+const { useDispatch, useSelect, select: directSelect } = Data;
+
+const AccountCreate = () => {
+	const { siteName, siteURL } = global.googlesitekit.admin;
```

Comment:
I'd say we should add it to the `core/site` store as part of this issue like we did with `isUsingProxy`.

Reply:
Sure, I can add that. @aaemnnosttv then would we use `useSelect` here as well or can we use `select` and move this outside the component function?

Reply:
> @aaemnnosttv then would we use useSelect here as well or can we use select and move this outside the component function?

Yes, we should always use `useSelect` so that the component updates properly if the value isn't immediately available. In this case, it should be, but we should avoid assuming that.

E.g. https://github.com/google/site-kit-wp/blob/f9d9806a6dc860dd64a5672ebcc802c584479374/assets/js/modules/analytics/common/anonymize-ip-switch.js#L40

Just to clarify, we should never use `Data.select` directly, even though it will be the same as what is passed to `useSelect` in production. One aspect is related to how hooks work and making sure the component updates with the datastore, but the other is related to tests. In tests, we create a fresh registry and register the stores on it all the time so in this case `registry != Data`. If you need access to the registry specifically, there is a `useRegistry` hook for inside a component, otherwise from within the datastore we always use `Data.commonActions.getRegistry` which yields the current registry (same with `createRegistrySelector` and `createRegistryControl`).

Reply:
Thanks for clarifying, now using `useSelect` everywhere and will here as well.

Reply:
added and used blog name from state in https://github.com/google/site-kit-wp/pull/1389/commits/83ea845f8ea1ea7a9b31e1959181934bdbb1aa17